lib/shared/puppet_forge/version.rb

```diff
@@ -0,0 +1,3 @@
+module PuppetForge
+  VERSION = '2.0.0dev'
```

Comment:
Looks like the rubygems pattern is a dot seperator: http://guides.rubygems.org/patterns/#prerelease-gems